pkgs/tools/networking/unbound/default.nix

```diff
@@ -37,7 +37,8 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ makeWrapper ];
 
-  buildInputs = [ openssl nettle expat libevent ]
+  propagatedBuildInputs = [ libevent ];
```

Comment:
You are sure about this change?

Reply:
The unbound pkgconfig says:
Requires.private:   libcrypto libssl libevent

Maybe its pkgconfig is wrong (meant for static lib). Likely I already had openssl and I missed simply libevent here, so I added it.
Should I add openssl to propagated? I don't want to patch its pkgconfig.

Reply:
for dynamically linked programs, the `.private` libs should not be propagated. Please put libevent back into `buildInputs`

Reply:
Aah right. I suspected that. In fact today I have no idea what was that change about... I prepared the change long before the PR for my own use and likely it's completely unrelated to oxen. I'll take it out!